src/components/Settings/SettingsAddressbookShare.vue

```diff
@@ -145,12 +147,12 @@ export default {
 					let matchingUsers = Object.values(response[0].data.ocs.data.users)
 					let matchingGroups = response[1].data.ocs.data.groups
 					try {
-						this.formatMatchResults(matchingUsers, query, false)
+						debounce(this.formatMatchResults(matchingUsers, query, false), 250, true)
```

Comment:
I used 500 in the details, but this is a different behavior, so 250 is also fine :)